src/ManageCourses.Domain/Models/McUser.cs

```diff
@@ -41,6 +41,8 @@ public class McUser : McBase
         /// When the user was sent the welcome email
         /// </summary>
         public DateTime? WelcomeEmailDateUtc { get; set; }
+        
+        public DateTime? AcceptTermsDateUtc { get; set; }
```

Comment:
Naming suggestion: `DateAcceptedTermsUtc`
I think it reads better and is clearer in its meaning even though it's not consistent.

minor.

Reply:
not in remit of changed code, card it up